tests/cypress/e2e/form/question_types/item.cy.js

```diff
@@ -0,0 +1,124 @@
+/**
+ * ---------------------------------------------------------------------
+ *
+ * GLPI - Gestionnaire Libre de Parc Informatique
+ *
+ * http://glpi-project.org
+ *
+ * @copyright 2015-2024 Teclib' and contributors.
+ * @copyright 2003-2014 by the INDEPNET Development Team.
+ * @licence   https://www.gnu.org/licenses/gpl-3.0.html
+ *
+ * ---------------------------------------------------------------------
+ *
+ * LICENSE
+ *
+ * This file is part of GLPI.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ * ---------------------------------------------------------------------
+ */
+
+describe('Item form question type', () => {
+    beforeEach(() => {
+        cy.createWithAPI('Glpi\\Form\\Form', {
+            'name': 'Tests form for the item form question type suite',
+        }).as('form_id');
+
+        cy.createWithAPI('Ticket', {
+            'name': 'Test ticket',
+        }).as('ticket_id');
+
+        cy.login();
+        cy.changeProfile('Super-Admin', true);
+
+        cy.get('@form_id').then((form_id) => {
+            const tab = 'Glpi\\Form\\Form$main';
+            cy.visit(`/front/form/form.form.php?id=${form_id}&forcetab=${tab}`);
+
+            // Add a new question
+            cy.findByRole("button", { name: "Add a new question" }).should('exist').click();
+
+            // Set the question name
+            cy.findByRole("textbox", { name: "Question name" }).should('exist').type("Test item question");
+
+            // Store the question section
+            cy.findByRole("option", { name: "New question" }).should('exist').as('question');
+
+            // Change question type
+            cy.findByRole("combobox", { name: "Short answer" }).should('exist').select("Item");
+        });
+    });
+
+    it('test defining new ticket as default value', () => {
+        // Click on the itemtype dropdown
+        cy.findByRole("combobox", { name: "-----" }).click();
```

Comment:
A new `getDropdownByLabelText` command was merged this morning to be able to get select2 dropdown by their label names instead of their current values (like we do for all others inputs).

Could you rebase and use it ?